packages/compartment-mapper/src/policy.js

```diff
@@ -0,0 +1,305 @@
+// @ts-check
+
+/** @typedef {import('./types.js').PolicyIdentityComponents} PolicyIdentityComponents */
+
+const { create, entries, assign, keys, freeze } = Object;
+const { stringify } = JSON;
+
+const copyGlobals = (globals, list) => {
+  const g = create(null);
```

Comment:
Let’s not lose moral ground over mathematicians and instead use descriptive variable names.

Reply:
That's the last one I missed. Noticed it in the meeting yesterday.